<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverheadsInfoView</class>
 <widget class="QWidget" name="OverheadsInfoView">
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="lblNowOverhead">
     <property name="text">
      <string>now() overhead:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="lblNowOverheadValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblLoopOverhead">
     <property name="text">
      <string>Loop overhead:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lblLoopOverheadValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblVirtualOverhead">
     <property name="text">
      <string>VCall overhead:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="lblVirtualOverheadValue">
     <property name="text">
      <string>N/A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
